src/Surfnet/ServiceProviderDashboard/Application/Service/EntityServiceInterface.php

```diff
@@ -35,10 +35,10 @@ public function createEntityUuid();
     /**
      * @param string $id
      * @param string $manageTarget
-     * @param int $serviceId
+     * @param Service $serviceId
      * @return mixed
      */
-    public function getEntityByIdAndTarget($id, $manageTarget, $serviceId);
+    public function getEntityByIdAndTarget($id, $manageTarget, Service $serviceId);
```

Comment:
Please also rename the parameter name? As it no longer is just the serviceId.

Reply:
Good catch